### PR TITLE
Adding ESRI Ocean tile layers

### PR DESCRIPTION
**Pull request by rsignell-usgs**
Adding ESRI Ocean Base and Ocean Reference layers

### CODE REVIEW

**Comment by philippjfr** [comment]
This is great thanks!

**Comment by rsignell-usgs** [comment]
@philippjfr or @jbednar These layers need to be tested and please check the attributions -- I wasn't quite sure how those worked.

**Comment by rsignell-usgs** [comment]
I just tested and they work okay, but I'm still not sure about whether I did the [attributions](https://github.com/pyviz/geoviews/blob/91d397fa6c8b96ed35d6d0385badf61fd5c42bec/geoviews/tile_sources.py#L39-L46) correctly.

**Comment by philippjfr** [comment]
Yes, that all looks correct to me.

**Comment by philippjfr** [comment]
The attributions code isn't great tbh, but it does the job so far.

File: geoviews/tile_sources.py [hunk 1]
```diff
@@ -69,7 +77,9 @@
 EsriNatGeo = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/NatGeo_World_Map/MapServer/tile/{Z}/{Y}/{X}', name="EsriNatGeo")
 EsriUSATopo = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/USA_Topo_Maps/MapServer/tile/{Z}/{Y}/{X}', name="EsriUSATopo")
 EsriTerrain = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/World_Terrain_Base/MapServer/tile/{Z}/{Y}/{X}', name="EsriTerrain")
-EsriReference = WMTS('http://server.arcgisonline.com/ArcGIS/rest/services/Reference/World_Reference_Overlay/MapServer/tile/{Z}/{Y}/{X}', name="EsriReference")
+EsriReference = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/Reference/World_Reference_Overlay/MapServer/tile/{Z}/{Y}/{X}', name="EsriReference")
+EsriOceanBase = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Base/MapServer/tile/{Z}/{Y}/{X}', name="EsriTerrain")
+EsriOceanReference = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Reference/MapServer/tile/{Z}/{Y}/{X}', name="EsriReference")
```
Comment (philippjfr) [inline geoviews/tile_sources.py]:
```suggestion
EsriOceanReference = WMTS('https://server.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Reference/MapServer/tile/{Z}/{Y}/{X}', name="EsriOceanReference")
```

**Comment by philippjfr** [comment]
I requested one small change otherwise happy to merge.

**Comment by philippjfr** [comment]
Thanks again!

**Pull request merged**